src/components/App.tsx

```diff
@@ -1,3 +1,4 @@
+import * as A from "@automerge/automerge"
```

Comment:
should be importing from `next`?

Reply:
per pvh it doesn't matter for the methods we're using. but as a general practice in TEE i think it's good because then we don't accidentally use old APIs in other places where we deal w text